### PR TITLE
plan: let getPushedProp fit the plan with multi children.

### PR DESCRIPTION
**Pull request by hanfei1991**
in old logic, `getPushedProp` will always process only one child. We refine it in order to deal with multiple children.

@zimulala @winoros PTAL

### CODE REVIEW

**Comment by winoros** [comment]
LGTM

File: plan/plan.go [hunk 1]
```diff
@@ -263,7 +263,7 @@ type PhysicalPlan interface {
 	ToPB(ctx context.Context) (*tipb.Executor, error)
 
 	// getPushedProp tries to push the required property to its child and get the result property.
```
Comment (zimulala) [inline plan/plan.go]: child -> children?
Update this comment.
And this function name may also need to update.

**Comment by hanfei1991** [comment]
@zimulala PTAL

File: plan/plan.go [hunk 2]
```diff
@@ -262,8 +262,8 @@ type PhysicalPlan interface {
 	// ToPB converts physical plan to tipb executor.
 	ToPB(ctx context.Context) (*tipb.Executor, error)
 
-	// getPushedProp tries to push the required property to its child and get the result property.
-	getPushedProp(prop *requiredProp) []*requiredProp
+	// getChildrenPossibleProps tries to push the required properties to its children and return all the possible property.
```
Comment (zimulala) [inline plan/plan.go]: s/property/properties

**Comment by zimulala** [comment]
Rest LGTM

**Comment by hanfei1991** [comment]
@zimulala PTAL

**Comment by zimulala** [comment]
LGTM

**Pull request merged**